### DISCUSSION TITLE
关于导入记录

### DISCUSSION THREAD

**Discussion by Yanshao-ria** [lgou2w/HoYo.Gacha#24] [Q&A] [answered]
请问有没有导入功能，有符合UIGF数据格式的json格式抽卡记录想要导入

**Comment by lgou2w** [comment, marked as answer]
添加指定游戏账号后。在 `祈愿` 或 `跃迁` 页面的顶部右上角工具栏，点击第一个图标 `导入记录` 选择符合格式的 JSON 文件。

**Comment by Yanshao-ria** [comment]
谢谢，已解决




檐少ria
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "lgou2w/HoYo.Gacha"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月7日(星期天) 凌晨0:02
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [lgou2w/HoYo.Gacha] 关于导入记录 (Discussion #24)





 
添加指定游戏账号后。在 祈愿 或 跃迁 页面的顶部右上角工具栏，点击第一个图标 导入记录 选择符合格式的 JSON 文件。
